vars/PipelineNode.groovy

```diff
@@ -107,15 +107,7 @@ def call(body) {
 
       // start of Documentation stage
       stage('Documentation') {
-        if(config.documentation) {
-          createNodeComposeDocsEnv(config, './documentation.json')
-          sh(script: "docker-compose -f documentation.json up --no-start")
-          sh(script: "docker-compose -f documentation.json run main npm run docs")
-          createNodeDocumentationGcsBucket(config)
-          uploadNodeDocumentation(config)
-        } else {
-          echo("skipping documentation.")
-        }
+        echo("skipping documentation.")
```

Comment:
Má nějaký význam tam tu stage vůbec mít? Ta hláška `skipping documentation.` mi přijde celkem matoucí - vypadalo by to, že tam ta stage je a jen je nakonfigurované její skipnutí...

Reply:
OK, smazáno